README.md

```diff
@@ -1,3 +1,45 @@
 # Northcoders News API
 
-For instructions, please head over to [L2C NC News](https://l2c.northcoders.com/courses/be/nc-news).
+## Setup Instructions
+
+### Setting up the Project
+
+1. **Clone the Repository:**
+git clone <YOUR_CLONED_REPO_URL>
+
+2. **Create a New Public GitHub Repository:**
+- Create a new public repository on GitHub without initializing it with a readme, .gitignore, or license.
+- Copy the repository URL.
+
+3. **Push Code to the New Repository:**
+- Navigate to the cloned local version of the project.
+- Run the following commands:
+  ```
+  git remote set-url origin <YOUR_NEW_REPO_URL>
+  git branch -M main
+  git push -u origin main
+  ```
+
+### Creating the Databases
+
+1. **Set Up Environment Variables:**
+- You need to create two `.env` files: `.env.test` and `.env.development`.
+- Add `PGDATABASE=` to each file, with the correct database name for that environment (refer to `/db/setup.sql` for the database names).
```

Comment:
You could be more explicit here and give the names of the databases for the .env files in the readMe